dds_cli/__main__.py

```diff
@@ -1605,6 +1605,11 @@ def get_data(
             "Flag '--get-all' cannot be used together with options '--source'/'--source-path-fail'."
         )
         sys.exit(1)
+    elif not get_all and not (source or source_path_file):
+        LOG.error(
+            "Specify either '--source'/'--source-path-file' to download specific directories/files, or '--get-all' to download all project contents."
```

Comment:
Just a thought, do you think this will be interpreted as that `--source` is for directories and `--source-path-file` is for files? Maybe in that case we should have "directories or files" or something

Reply:
for me `directories/files` = `directories or files`, doesn't it?
It doesn't matter for me ... actually the `either` kind of requires `or`, like:

"Specify either '--source' or '--source-path-file' ... 

and then the second one can be either `/` or `or`

Reply:
Yeah it is but it kind of looked like the `--source/--source-path-file` was meant to match `directories/files`, or at least I could see how someone would interpret it like that. 

Yeah do the 
```
"Specify either '--source' or '--source-path-file' to download specific directories/files, or '--get-all' to download all project contents."
```

Reply:
yes, `/` on both places is straightforward, but the `either` doesn't go with the slashes. I am changing it.